lib/src/public_ext.dart

```diff
@@ -88,7 +88,7 @@ extension BuildContextEasyLocalizationExtension on BuildContext {
   Locale get locale => EasyLocalization.of(this).locale;
 
   /// Change app locale
-  set locale(Locale val) => EasyLocalization.of(this).locale = val;
+  void setLocale(Locale val) async => EasyLocalization.of(this).setLocale(val);
```

Comment:
I think it's better to keep this `set locale(Locale val)` with  `@Deprecated()` msg, In order to avoid breaking change like this :
```suggestion
  /// New API Change app locale
  void setLocale(Locale val) async => EasyLocalization.of(this).setLocale(val);

  /// Old Change app locale
  @Deprecated(
      'This is the func used in the old version of EasyLocalization. The modern func is `setLocale(val)` . '
      'This feature was deprecated after v4.0.0')
  set locale(Locale val) => EasyLocalization.of(this).setLocale(val);
```

Reply:
Yep

Reply:
Why keep something, if we are going for a major update?

Reply:
@spiritinlife  to avoid breaking change,